packages/@react-facet/dom-fiber/src/types.ts

```diff
@@ -138,6 +140,10 @@ export type ElementContainer = {
   text?: Unsubscribe
 }
 
+export const isElementContainer = (value: unknown): value is ElementContainer => {
+  return value != null && (value as ElementContainer).children != null
+}
```

Comment:
```suggestion
export const isElementContainer = (value: ElementContainer | TextContainer): value is ElementContainer => {
  return value != null && 'children' in value
}
```

Let's avoid the manual casting